external-crates/move/crates/move-compiler/tests/development/enums/matching/struct_visibility.exp

```diff
@@ -0,0 +1,7 @@
+error[E04001]: restricted visibility
+   ┌─ tests/development/enums/matching/struct_visibility.move:13:13
+   │
+13 │             m::S { t } => t,
+   │             ^^^^^^^^^^ Invalid pattern for '0x42::m::S'.
+ All struct can only be matched in the module in which they are declared
```

Comment:
```suggestion
 All structs can be matched only in the module in which they are declared
```

Reply:
"can only be X" is consistent with wording across the rest of the compiler, e.g.,

- "Invalid instantiation of '{}::{}'.\nAll structs can only be constructed in"
- "Non-u64 constants can only be used as error codes if"
- "Lambdas can only be used directly as arguments"